src/app/components/common/NewWhiteWrapper/components/ValidatorsFlow/ValidatorsFlow.tsx

```diff
@@ -32,7 +32,7 @@ const ValidatorsFlow = (props: Props) => {
           <Grid item className={classes.BackNavigation} onClick={onNavigationClicked} />
           <Grid item className={classes.HeaderText}>{header}</Grid>
           <Grid item className={classes.subHeaderText}>|</Grid>
-          <Grid item className={classes.subHeaderText}>{longStringShorten(clusterStore.getClusterHash(cluster.operators), 4)}</Grid>
+          <Grid item className={classes.subHeaderText}>{longStringShorten(clusterStore.getClusterHash(cluster.operators).slice(2), 4)}</Grid>
```

Comment:
Use `.replace(/^0x/g, '');`
Because it should be used in more than 1 place, move it to `longStringShorten` function.
Add to it one extra-parameter with name `replacements: Record<RegEx | string, string>` with default value of `null`
And inside of function iterate over dictionary and do `replace(key, value)`.
Also, do these replacements `before` shortening the string, so that after shortening it would have correct lenght

Reply:
done